mlflow/utils/model_utils.py

```diff
@@ -286,20 +286,10 @@ def _add_code_from_conf_to_system_path(local_path, conf, code_key=FLAVOR_CONFIG_
     """
     assert isinstance(conf, dict), "`conf` argument must be a dict."
 
-    modules_to_reload = set()
     if code_key in conf and conf[code_key]:
         code_path = os.path.join(local_path, conf[code_key])
         _add_code_to_system_path(code_path)
 
-        for loaded_module in sys.modules:
-            if os.path.exists(os.path.join(code_path, loaded_module.replace('.', os.sep))):
-                # this is a module loaded from code path
-                modules_to_reload.add(loaded_module)
-
-    # invalidate these modules cache, to ensure they are reloaded when loading model.
-    for module in modules_to_reload:
-        sys.modules.pop(module)
```

Comment:
This causes strange error in `recipe` tests, need more investigation. I will make this change in followup PR.

Reply:
What is `strange error`?

Reply:
This is import important, isn't it? Can't we really comment this out?

Reply:
Let me repro the error in https://github.com/mlflow/mlflow/pull/11989 CI output
I can't repro it locally.